node/clients/index.ts

```diff
@@ -1,13 +1,19 @@
+import type { IOContext } from '@vtex/api'
 import { IOClients } from '@vtex/api'
 
-import { GraphQLServer } from './graphqlServer'
-import { Checkout } from './Checkout'
 import { AuthUser } from './AuthUser'
+import { Checkout } from './Checkout'
+import { OrganizationsGraphQLClient } from './Organizations'
+import StorefrontPermissions from './StorefrontPermissions'
 
 // Extend the default IOClients implementation with our own custom clients.
 export class Clients extends IOClients {
-  public get graphQLServer() {
-    return this.getOrSet('graphQLServer', GraphQLServer)
```

Comment:
Just making sure: there's no remaining call sites for this client that was removed, right?

Reply:
Nope! I don't know, but the client was created generically, specifying the graphql-server instead of the app that provides the graphql operations. So, for it to work, I needed to split the client into two, one for each app provider.